main/reference/zoe-api/zoe-data-types.md

```diff
@@ -127,11 +133,33 @@ const { quoteAmount, quotePayment } = priceQuote;
 just like **[Amounts](/reference/ertp-api/ertp-data-types.md#amount)**.
 A **Ratio** cannot have a denominator value of 0.
 
-The ratio functions have to be imported.
-
 The most common kind of **Ratio** is applied to an **Amount** of a particular **Brand**
 and produces results of the same **Brand**.
 
 **Ratios** can also have two different **Brands**, essentially typing them such as miles per
-hour or US dollars for Swiss francs (an exchange rate ratio).
-
+hour or US dollars for Swiss francs (i.e., an exchange rate ratio).
+
+## TransferPart
+
+**TransferParts** are the individual elements of the *transfer* array passed into the 
+**[atomicRearrange()](./zoe-helpers.md#atomicrearrange-zcf-transfers)** function. Each **TransferPart**
+represents one or two **[Allocation](#allocation)** changes among existing 
+**[ZCFSeats](./zcfseat.md)**. Each **TransferPart** consists of 4 fields, none of which are mandatory:
```

Comment:
I think we should re-word this. AIUI, Javascript arrays have *elements*, not *fields*.  I've asked MarkM what the appropriate terminology would be.

Reply:
Aw, man... JavaScript calls those "elements"? That's awful terminology; I associate "elements" with "HTML elements", which is pretty misleading.

Well, JavaScript is gonna JavaScript. I've changed "fields" to "elements".